js/views/modals/listingDetail/Listing.js

```diff
@@ -278,14 +300,24 @@ export default class extends BaseModal {
 
   get $photoSection() {
     return this._$photoSection ||
-      (this._$photoSection = this.$('#photoSection'));
+      (this._$photoSection = this.$('.js-photoSection'));
   }
 
   get $photoSelected() {
     return this._$photoSelected ||
       (this._$photoSelected = this.$('.js-photoSelected'));
   }
 
+  get $photoSelectedHeight() {
```

Comment:
Your caching both $photoSelectedHeight() & $photoSelectedWidth(), but you're never clearing `_$photoSelectedHeight` and `._$photoSelectedWidth` on render. So, on a re-render that will be cached to an inaccurate value.

I don't think it necessary to cache these guys since you're already caching the $photoSelected dom element, and beyond getting the element, I don't think determining height and width makes any special trips to the DOM or it's a particularly intensive process.